articles/defender-for-cloud/upcoming-changes.md

```diff
@@ -45,6 +46,27 @@ If you're looking for the latest release notes, you can find them in the [What's
 | [Deprecating two security incidents](#deprecating-two-security-incidents) |  | November 2023 |
 | [Defender for Cloud plan and strategy for the Log Analytics agent deprecation](#defender-for-cloud-plan-and-strategy-for-the-log-analytics-agent-deprecation) |  | August 2024 |
 
+## General Availability of Unified Disk Encryption recommendations
+
+**Announcement date: March 28, 2024**
+
+**Estimated date of change: April 30, 2024**
+
+Unified Disk Encryption recommendations will be released for General Availability (GA) within Azure Public Cloud in April 2024. The recommendations enable customers to audit encryption compliance of virtual machines with Azure Disk Encryption or EncryptionAtHost.
+
+**Recommendations moving to GA:**
+
+| Recommendation DisplayName | Assessment key |
+| ---- | ---- |
+| Linux virtual machines should enable Azure Disk Encryption or EncryptionAtHost | a40cc620-e72c-fdf4-c554-c6ca2cd705c0 |
+| Windows virtual machines should enable Azure Disk Encryption or EncryptionAtHost | 0cb5f317-a94b-6b80-7212-13a9cc8826af |
+
+Azure Disk Encryption (ADE) and EncryptionAtHost provide encryption at rest coverage, as described in [Overview of managed disk encryption options - Azure Virtual Machines | Microsoft Learn](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption-overview), and we recommend enabling either of these on virtual machines. 
+
+The recommendations depend on [Guest Configuration](https://learn.microsoft.com/en-us/azure/governance/machine-configuration/overview). Pre-requisites to onboard to Guest configuration should be enabled on virtual machines for the recommendations to complete compliance scans as expected. 
```

Comment:
```suggestion
The recommendations depend on [Guest Configuration](/azure/governance/machine-configuration/overview). Pre-requisites to onboard to Guest configuration should be enabled on virtual machines for the recommendations to complete compliance scans as expected. 
```

Reply:
Thanks for the review. Updated the links in latest iteration.